simplivity/resources/backups.py

```diff
@@ -26,6 +26,18 @@ class Backups(ResourceBase):
     def __init__(self, connection):
         super(Backups, self).__init__(connection)
 
+    def delete_multiple_backups(self, backups, timeout=-1):
```

Comment:
Let's keep this method after the get_all method.

Reply:
I moved the code block below def get_all as requested